src/7-to-8/major-changes/platforms.rst

```diff
@@ -210,13 +210,20 @@ And the platform settings for these examples might be:
            # A computer with PBS, that takes local job submissions
            job runner = pbs
            hosts = localhost
+           install target = localhost
 
        [[slurm_supercomputer]]
            # This computer with Slurm requires you to use a login node.
            hosts = login_node01, login_node02  # Cylc will pick a host.
            job runner = slurm
 
 
+Note that in these examples, it is assumed that ``linuxboxNN``, ``pbs_local`` and
+``slurm_supercomputer`` have distinct file systems.
+If platforms share a file system they must have their :term:`install target`
+configured, this should be done at site level. For more information see
+:ref:`Install Targets`.
```

Comment:
```suggestion
If an install target is not set, a platform will use its own platform name
as the install target name. If multiple platforms share a file system
and :ref:`install target <Install Targets>` is not set each will carry out a separate (slow) remote
install procedure. Sets of platforms which share a file system should specify
a single :ref:`install target <Install Targets>`.
```

Reply:
If they share a file system the authentication keys will clash (resulting in remote init error) if they have different install targets configured though?

Reply:
Ok, then you are right, they must. I still think that the clarification makes sense

```suggestion
Sets of platforms which share a file system must specify
a single :ref:`install target <Install Targets>`.

.. note::
   If an install target is not set, a platform will use its own platform name
   as the install target name. If multiple platforms share a file system
   and :ref:`install target <Install Targets>` they will not work.
```